platforms/i2c/jhd1313m1_driver.go

```diff
@@ -145,6 +147,12 @@ func (h *JHD1313M1Driver) Home() (err error) {
 
 func (h *JHD1313M1Driver) Write(message string) (err error) {
 	for _, val := range message {
+		if val == '\n' {
+			if err = h.SetPosition(16); err != nil {
+				return
+			}
+			continue
+		}
 		if err = h.connection.I2cWrite(h.lcdAddress, []byte{0x40, byte(val)}); err != nil {
```

Comment:
nitpick, shoudn't we use `LCD_2NDLINEOFFSET` instead of `0x40` in the `I2cWrite` call?